schemas/performance-artifact.json

```diff
@@ -80,7 +80,7 @@
           "title": "Extra options used in running suite",
           "items": {
             "type": "string",
-            "maxLength": 45
+            "maxLength": 100
```

Comment:
We could keep the old `"maxLength": 45` & add a more explicit [additionalItems](https://json-schema.org/understanding-json-schema/reference/array.html#items). Something like:
```json
"additionalItems": {
  "type": "string", 
  "maxLength": 100
}
```

Reply:
I did some tests and apparently, it cannot be done.
If I use 
```json
            "items": [
             {
           	"type": "string",
            	"maxLength": 100
          	},
            {
            	"type": "string",
            	"maxLength": 45
          	},
```
then the order of the list elements will be enforced, and this is not ok.
JSON schema array's are [ordered](https://json-schema.org/understanding-json-schema/reference/array.html?#array).

Reply:
Ok, then. Let's keep it as it is.